cfme/tests/cloud_infra_common/test_policy_simulation.py

```diff
@@ -0,0 +1,79 @@
+# -*- coding: utf-8 -*-
+"""This module tests events that are invoked by Cloud/Infra VMs."""
+import pytest
+
+from cfme.cloud.provider import CloudProvider
+from cfme.common.vm_views import PolicySimulationDetailsView
+from cfme.common.vm_views import PolicySimulationView
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+pytestmark = [
+    pytest.mark.usefixtures('uses_infra_providers', 'uses_cloud_providers', 'setup_provider'),
+    pytest.mark.tier(2),
+    pytest.mark.provider(
+        classes=[CloudProvider, InfraProvider],
+        selector=ONE_PER_CATEGORY,
+        scope="module"
+    ),
+]
+
+
+@pytest.fixture
+def vm_and_filter(appliance, provider):
+    collection = provider.appliance.provider_based_collection(provider)
+    vm = collection.all()[0]
+    filter = {'name': vm.name, 'provider': provider}
+    yield vm, filter
```

Comment:
@john-dupuy  
You can change ```filter``` variable to some other name. Because, it is **python keyword**.